fix_includes.py

```diff
@@ -1815,9 +1886,8 @@ def _DecoratedMoveSpanLines(iwyu_record, file_lines, move_span_lines, flags):
   if reorder_span is None:     # must be a new #include we're adding
     # If we're a forward-declare inside a namespace, see if there's a
     # reorder span inside the same namespace we can fit into.
-    if kind == _FORWARD_DECLARE_KIND:
```

Comment:
What happened to this check for `_FORWARD_DECLARE_KIND`? I'll admit I don't fully understand what's going on here... Does `_GetNamespaceLevelReorderSpans` somehow cover for this now?

Reply:
This was actually an oversight on my part.  I must've removed it by accident.  It didn't change the way the script worked but adding it back should greatly reduce the calls made to _GetNamespaceLevelReorderSpans